### DISCUSSION TITLE
[BUG]在线升级

### DISCUSSION THREAD

**Discussion by iygwei** [1Panel-dev/1Panel#6586] [Q&A]
### 1Panel 版本

1.10.16-lts

### 问题描述

点升级会报请求超时,但是我把域名在浏览器访问是正常的xxxx//api/v1/settings/upgrade

### 重现步骤

/api/v1/settings/upgrade

**Comment by wanghe-fit2cloud** [comment]
1Panel 的版本信息统一维护在阿里云 OSS 中，建议尝试切换一下 1Panel 服务器的 DNS。

**Reply by iygwei** [reply]
> 1Panel 的版本信息统一维护在阿里云 OSS 中，建议尝试切换一下 1Panel 服务器的 DNS。

怎么切换

**Reply by iygwei** [reply]
> 1Panel 的版本信息统一维护在阿里云 OSS 中，建议尝试切换一下 1Panel 服务器的 DNS。

我发现了 我如果科学上网就通了 你这服务器是加到国外去了吗

**Reply by iygwei** [reply]
好像你们是接受的时间超过30s就算它超时了
